wgengine/magicsock/magicsock.go

```diff
@@ -1318,7 +1318,14 @@ func (c *Conn) sendDiscoMessage(dst netip.AddrPort, dstKey key.NodePublic, dstDi
 		// Can't send. (e.g. no IPv6 locally)
 	} else {
 		if !c.networkDown() {
-			c.logf("magicsock: disco: failed to send %v to %v: %v", disco.MessageSummary(m), dst, err)
+			if isPMTUError(err) {
+				metricSentDiscoPeerMTUErrors.Add(1)
```

Comment:
It's not an error necessarily, just a signal from the host stack that the message is too big from what was already discovered in the host stack's implementation of PMTU, not to be confused with what we call PMTU here.